interface/billing/era_payments.php

```diff
@@ -247,69 +247,74 @@ function OnloadAction()
-                                <br />
-                                <!--onKeyUp="ajaxFunction(event,'non','search_payments.php');"-->
-                                <div id='ajax_div_insurance_section'>
-                                    <div id='ajax_div_insurance_error'></div>
-                                    <div id="ajax_div_insurance" style="display:none;"></div>
+                            <div class="row">
+                                <div class="form-group col-6">
+                                    <label class="control-label" for="type_code"><?php echo xlt('Insurance'); ?>:</label>
+                                    <input id="hidden_ajax_close_value" type="hidden" value="<?php echo attr($type_code); ?>" />
+                                    <input autocomplete="off" class="form-control" id='type_code' name='type_code' onkeydown="PreventIt(event)"  type="text" value="<?php echo attr($type_code); ?>" />
+                                    <br />
+                                    <!--onKeyUp="ajaxFunction(event,'non','search_payments.php');"-->
+                                    <div id='ajax_div_insurance_section'>
+                                        <div id='ajax_div_insurance_error'></div>
+                                        <div id="ajax_div_insurance" style="display:none;"></div>
+                                    </div>
+                                </div>
+                                <div class="form-group col-6">
+                                    <label class="control-label" for="div_insurance_or_patient"><?php echo xlt('Insurance ID'); ?>:</label>
+                                    <div class="form-control" id="div_insurance_or_patient" >
+                                        <?php echo text($hidden_type_code); ?>
+                                    </div>
+                                    <input id="description" name="description" type="hidden" />
                                 </div>
                             </div>
-                            <div class="form-group col-3">
-                                <label class="control-label" for="div_insurance_or_patient"><?php echo xlt('Insurance ID'); ?>:</label>
-                                <div class="form-control" id="div_insurance_or_patient" >
-                                    <?php echo text($hidden_type_code); ?>
+                            <!-- can change position of buttons by creating a class 'position-override' and adding rule text-align:center or right as the case may be in individual stylesheets -->
+                            <div class="form-group mt-3">
+                                <div class="col-sm-12 text-left position-override">
+                                    <div class="btn-group" role="group">
+                                        <a class="btn btn-primary btn-save" href="#" onclick="javascript:return Validate();"><?php echo xlt('Process ERA File');?></a>
```

Comment:
This has incorrect padding/spacing on the left side of this button

Also the navbar needs to be fixed

Reply:
So, should I remove the padding or margin?

Reply:
Navbar padding on this page, new_payment.php, and search_payments.php should be set to `p-2` instead of `p-4`